api/cmd/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/project-nova/backend/pkg/database"
+	"github.com/thirdweb-dev/go-sdk/v2/thirdweb"
+)
+
+type WhitelistWallet struct {
+	ID          string `gorm:"primaryKey;column:id"`
+	Address     *string
+	MerkleProof *string
+	CreatedAt   time.Time
+}
+
+type StoryNFT struct {
+	Name   string
+	Symbol string
+	Count  uint64
+	URI    string
+}
+
+func (WhitelistWallet) TableName() string {
+	return "whitelist_wallet"
+}
+
+func main() {
+	r := gin.Default()
+
+	r.GET("/mint/proof", func(c *gin.Context) {
+		address := c.DefaultQuery("address", "")
+
+		// Validate address
+		if address == "" {
+			c.String(http.StatusBadRequest, fmt.Sprintf("input address is invalid, address: %s", address))
+			return
+		}
+
+		db, err := database.NewGormDB("postgresql://postgres:@localhost:23005/postgres?sslmode=disable")
```

Comment:
minor: use a config file

Reply:
Yes. Will do in next batch

Reply:
Added the config files for different envs and config flow